android/app/build.gradle

```
@@ -125,6 +125,32 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  * and the benefits of using Hermes will therefore be sharply reduced.
  */
 def enableHermes = project.ext.react.get("enableHermes", false);
+def envProperties = new Properties()
+def envPropertiesFile = rootProject.file('env.properties')
```

Er det mulig at denne heller kan lese av "root" sin `.env`? Virker ikke til at den må holdes i sync? Dvs
```suggestion
def envPropertiesFile = rootProject.file('../../.env')
```